stdlib/public/Backtracing/SymbolicatedBacktrace.swift

```diff
@@ -23,6 +23,11 @@ import Swift
 
 @_implementationOnly import OS.Libc
 @_implementationOnly import Runtime
+// Because we've turned off the OS/SDK modules, and we don't have a module for
+// stddef.h, and we sometimes build with -fbuiltin-headers-in-system-modules for
+// vfs reasons, stddef.h can be absorbed into a random module. Sometimes it's
+// SwiftOverlayShims.
```

Comment:
> Sometimes it's SwiftOverlayShims.

This sounds stochastic. Is there a way to give it a stable home regardless of flags?

Reply:
Backtracing would need to stop disabling intrinsic module maps so that it could use the OS/SDK module maps where stddef.h lives. I'm not really sure why it does that. I guess Backtracing could add more comprehensive module maps to cover the actual headers instead of assuming that Libc.h will absorb all of the effectively non-modular OS/SDK headers into Backtracking's OS.Libc module.

Reply:
It does it because the `Darwin` module (and its counterparts on Linux and Windows) are built later on in the build process than the backtracer, and attempting to fix that created dependency cycles in the CMake build system that were difficult to fix.

Reply:
I'd rather it wasn't the cause of random behaviour though; if we can avoid this by fixing the `OS.Libc` module map then I'd be in favour of doing that instead.

Reply:
The Swift overlays get built later, but does that preclude us from using the clang modules? Fixing this in `OS.Libc` would amount to copying the module maps we vfs overlay on Linux/Windows, and copying the Apple module maps. I think it would be better to just use them.

But the behavior has always been random with the way Backtracing is set up, I would prefer to just add this import here to keep Linux limping along, and do a better fix later.

Reply:
There is an additional problem that _mainly_ affects Darwin, namely that some of the definitions in the C headers are written in a manner that precludes their use from Swift, so I don't think it's _quite_ as simple as "just use the system module maps". I _did_ try that approach before deciding that it was better to do what it currently does. I have no objection to trying to use the system ones again, in principle, but it didn't work before.

Reply:
Seeing as how this isn't making the problem any worse, it's just landing on a different chaos number, could we possibly just go with this for now and try to fix it for real later?

Reply:
Seems good to me.

Reply:
Filed rdar://119565504 (amongst the other ones I mentioned) to take a new look at this.